micr_XPCTzarr/participants.tsv

```diff
@@ -0,0 +1,2 @@
+participant_id	sex	age	species	weight	height	institute	medical_information
+sub-LADAF-2020-31	F	69	homo sapiens	40	145	Laboratoire d'Anatomie des Alpes Françaises	type 2 diabetes, pelvic radiation to treat cancer of the uterus, right colectomy (benign lesion on histopathology), bilateral nephrostomy for acute obstructive renal failure, cystectomy, omentectomy and peritoneal carcinoma with occlusive syndrome
```

Comment:
Subject labels/participant IDs must be alphanumeric. e.g. `sub-LADAF202031`. The hyphens inside the label are not permitted.

Reply:
Thanks, I will update the `participant_id`! To comply with our internal naming conventions, I'd go with an extra `name` or `pseudo` and change `participant_id` into `01`, as I did with the `ses-01` (where the 'unique key' is [`proposal_reference` & `acq_time`]): do you think that is a relevant choice, @effigies ?

Reply:
Sure, that seems reasonable.